userspace/libscap/CMakeLists.txt

```diff
@@ -40,7 +40,9 @@ if(CMAKE_SYSTEM_NAME MATCHES "Linux")
 	endif()
 
     if(NOT DEFINED DRIVER_VERSION)
-        set(DRIVER_VERSION "${FALCOSECURITY_LIBS_VERSION}")
+        message(ERROR 
+			"No driver version set. Please either build the project from a git working directory or explicitly set the DRIVER_VERSION cmake cache entry."
+		)
```

Comment:
Shouldn't we stop the build if `DRIVER_VERSION` is not set? Could this end up with a driver using an invalid version?

Reply:
Never mind this, I'm still waking up 🤦🏻‍♂️